brainscore_language/metrics/linear_predictivity/test.py

```diff
@@ -4,7 +4,7 @@
 
 from brainio.assemblies import NeuroidAssembly
 from brainscore_language import load_metric
-from metric import linear_regression
+from . import linear_regression
```

Comment:
@aalok-sathe wanted to avoid relative plugins, he said they lead to issues on his end